### PR TITLE
Feature/is filled method

### PR DESCRIPTION
**Pull request by prevostjules**
Hi @Jabuf @Djoudi21 ,

As discussed in a previous PR, we were not 100% satisfied by the isFilled method since it was difficult to maintain.

The isFilled method is now a method contained in a super class called `UropaEntity`.
This isFilled method just compare the instance of a class (ex: `new Ceo({name: 'truc'})` with the instance of a class initialized only with the default values. 
We stringify both, compare them and, if they differ, it means it has been filled by the user. 
[Source](https://github.com/progressive-identity/uropa-ui/pull/76/files#diff-e13b56485244b3a07967cc17827f1ad8ffe3515d98012498ac88b8109affb61b)

Happy review

### CODE REVIEW

File: src/data/template/UropaEntity.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export class UropaEntity {
```
Comment (Jabuf) [inline src/data/template/UropaEntity.ts:1]: I think this should be an abstract class.

File: src/data/template/UropaEntity.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  /**
+   * Check if an instance of a class has been filled
+   * To do so, we compare the instance with the default instance
+   */
+  isFilled(): boolean {
```
Comment (Jabuf) [inline src/data/template/UropaEntity.ts:6]: I think we could do this quick improvement : 
```
isFilled(args = []): boolean {
    const defaultInstance = Reflect.construct(this.constructor, args)
```

**Review by Jabuf — APPROVED** [review]
Nicely done, it seems more efficient to do that for now. Also I love reflection, I always find that super fun to use.

File: src/data/template/UropaEntity.ts (hunk 2 above)
Comment (prevostjules) [reply src/data/template/UropaEntity.ts:6]: right we change that too ;)

**Pull request merged**